scss/pages/_pages.scss

```diff
@@ -0,0 +1,5 @@
+.page-civicrm-case-a { 
```

Comment:
This change is not described in the PR docs. Please add details.

Reply:
Added.